app/controllers/api/v1/admin/canonical_email_blocks_controller.rb

```diff
@@ -58,6 +58,8 @@ def set_canonical_email_blocks
   end
 
   def set_canonical_email_blocks_from_test
+    params.require(:email)
+
     @canonical_email_blocks = CanonicalEmailBlock.matching_email(params[:email])
```

Comment:
I know we have some parts of our codebase using this pattern, but I would just do:
```suggestion
    @canonical_email_blocks = CanonicalEmailBlock.matching_email(params.require(:email))
```